app/actions/CommonActions.scala

```diff
@@ -62,6 +62,29 @@ object CommonActions {
       case _ => _platform
     }
 
+    def playId: Option[String] = {
+      for {
+        playSessionCookie <- request.cookies.get("PLAY_SESSION")
+        value <- Option(playSessionCookie.value)
+      } yield {
+        value.take(value.indexOf("-"))
```

Comment:
I think `value.takeWhile(_ != '-')` would be clearer. Also, why are we only taking the characters up to the first `-`?

Reply:
Agree takeWhile(_ != '-') would be more elegant and clearer.

Reason we are only taking characters up to the first - is that - as far as I can tell - the id field isn't directly accessible. To get it you have to access the cookie value then strip out all the other fields that have been added to the session cookie - this is especially important as the main reason for this change is to store a unique identifier in the play session.